com.wonderlandengine.editor.yml

```diff
@@ -0,0 +1,45 @@
+app-id: com.wonderlandengine.editor
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: WonderlandEditor
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Enabling VR Headset connection over adb and GPU acceleration
+  - --device=all
+  # Needs to talk to the network for authentication
+  - --share=network
+  # We need access to external drives and the homefolder, or where ever the user might store assets.
+  - --filesystem=host
+  # For desktop notifications
+  - --talk-name=org.freedesktop.Notifications
+modules:
+
+  - name: glfw
+    buildsystem: cmake-ninja
+    build-options:
+      no-debuginfo: true
+    config-opts:
+      - -DBUILD_SHARED_LIBS=ON
+      - -DGLFW_BUILD_DOCS=OFF
+    sources:
+      - type: git
+        url: https://github.com/glfw/glfw.git
+        tag: 3.3.8
+        commit: 7482de6071d21db77a7236155da44c172a7f6c9e
+
+  - name: editor
+    buildsystem: simple
+    build-options:
+      no-debuginfo: true
+    build-commands:
+      - mkdir ${FLATPAK_DEST}/bin ${FLATPAK_DEST}/share
+      - mv bin/* ${FLATPAK_DEST}/bin
+      - mv lib/* ${FLATPAK_DEST}/lib
+      - mv share/* ${FLATPAK_DEST}/share
+    sources:
+      - type: archive
+        url: https://downloads.wonderlandengine.com/1.2.3/WonderlandEngine-1.2.3-Flatpak.tar.gz
```

Comment:
can this be built from source?

Reply:
No, the Wonderland Editor is proprietary software.